tests/EndToEndBuildTests/Directory.Build.props

```
@@ -5,6 +5,7 @@
     <RollForward>LatestMajor</RollForward>
   </PropertyGroup>
 
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <Import Project="$([MSBuild]::GetPathOfFileAbove('FSharpTests.Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

```suggestion
  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Only one of those 2 should stay, I think the lower one never does anything because such file is not there.

---

We have a weird dependency loop up [here](https://github.com/dotnet/fsharp/blob/main/Directory.Build.props#L92-L95). I tried removing it, then Cambridge tests blew up. I fixed those, things stopped compiling for Linux/MacOS.

So you're right in that it's a bad fix and only one thing should stay here, it also produces build warnings in E2E tests even locally. 

But that will unblock the CI, I'll try to address this later properly though.